cmake/aditof-setup.iss.cmakein

```diff
@@ -58,11 +53,11 @@ Name: "ukrainian"; MessagesFile: "compiler:Languages\Ukrainian.isl"
 Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"
 
 [Files]
-Source: "C:\projects\aditof-sdk\build\sdk\Release\aditof.dll"; DestDir: "{app}"
-Source: "C:\projects\aditof-sdk\build\examples\aditof-demo\Release\aditof-demo.exe"; DestDir: "{app}";
+Source: "D:\a\1\s\build_Release\sdk\Release\aditof.dll"; DestDir: "{app}"
```

Comment:
You can get the path to the build folder using cmake.

Reply:
Fixed.